README.md

```diff
@@ -0,0 +1,50 @@
+# TREF - Text Representation Evaluation Framework
+
+State-of-the-art Text Representations for Natural Language Processing tasks, an initial version of library focus on the Polish Language
+
+# Installation
```

Comment:
why not installation from pypi?

Reply:
I forgot ;p however we must check and build pypi package quite often. I got `ModuleNotFoundError: No module named 'typer'` after `pip install embeddings` command.

Reply:
but our package name is different try `pip install clarinpl-embeddings`

Reply:
Anyway we have already automatic pypi package building

Reply:
done